CONTRIBUTING.md

```diff
@@ -1,52 +1,121 @@
-# Contribute to the Confidential Containers project
-
-The Confidential Containers project is an open source project licensed under the
-[Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
-
-It comprises a number of repositories under the [GitHub Confidential
-Containers organisation](https://github.com/confidential-containers). Unless
-explicitly stated otherwise, all the Confidential Containers repositories follow the
-process documented here.
-
-## Code of Conduct
-
-All contributors must agree to the project [code of conduct](CODE_OF_CONDUCT.md).
-
-## Pull requests
-
-All the repositories accept contributions via [GitHub Pull requests (PR)](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests).
-Submit PRs by following the [GitHub workflow](#github-workflow).
-
-## GitHub basic setup
-
-To get started, complete the prerequisites below.
-
-### Prerequisites
-
-- Review [Contributor roles](#contributor-roles) that require special Git
-  configuration.
-
-- [Set up Git](https://help.github.com/en/github/getting-started-with-github/set-up-git).
-
-  > **Note:** The email address you specify must match the email address you
-  > use to sign-off commits.
-
-- [Fork and Clone](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the relevant repository at the
-  [Confidential Containers Project](https://github.com/confidential-containers).
-
-   Example: Your local clone should show `your-github-username`, as follows.
-   `https://github.com/${your-github-username}/community`.
-
-### Contributor roles
+# Contributor Guide
+
+So you want to contribute to Confidential Containers?
+This guide will get you up to speed on the mechanics of the project
+and offer tips about how to work with the community and make great
+contributions.
+
+First off, Confidential Containers (CoCo) is an open source project.
+You probably already knew that, but it's good to be clear
+that we welcome contributions, involvement, and insight from everyone.
+Just make sure to follow the [code of conduct](CODE_OF_CONDUCT.md).
+
+CoCo is licensed with [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
+
+This guide won't cover the architecture of the project, but you'll notice
+that there are many different repositories under the [CoCo
+organization](https://github.com/confidential-containers).
+A CoCo deployment integrates many different components.
+Fortunately the process for contributing to each of these subprojects
+is largely the same.
+There is one major exception. Some CoCo deployments leverage [Kata Containers](https://github.com/kata-containers),
```

Comment:
Which ones do not leverage Kata?

Reply:
I changed this to "most deployments" since enclave-cc is the only one that does not.